.github/workflows/lint.yml

```diff
@@ -16,10 +16,18 @@ jobs:
         with:
           persist-credentials: false
 
-      - uses: ./actions/setup-node-and-install-deps
```

Comment:
With `./actions/setup-node-and-install-deps`, npx playwright install is not executed, so it was removed.
I do not understand why.

Reply:
I have not investigated in depth because we have decided that RELEASE should be our priority.